Dockerfile

```diff
@@ -1,7 +1,12 @@
 FROM elixir:1.9.4-alpine as builder
 
 # Install SSL ca certificates
-RUN apk update && apk add ca-certificates && apk add bash
+RUN apk update && apk add ca-certificates bash
+RUN apk add \
+  --no-cache \
+  --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing \
+  --repository http://dl-cdn.alpinelinux.org/alpine/edge/main \
```

Comment:
The base image this is built upon is based on Alpine 3.11, not edge. Though edge does not contain `jo`.

Is it wise to add the edge overlays into a 3.11?

Reply:
Could we add a comment mentioning this? Just in case someone subsequently installs another package that isn't happy with the mixture of sources. 

Assuming `jo` is fine. I would imagine that any other packages should be installed _before_ line 5 (to be safe).